src/vsr/superblock.zig

```diff
@@ -545,30 +545,13 @@ comptime {
 /// The size of the entire superblock storage zone.
 pub const superblock_zone_size = superblock_copy_size * constants.superblock_copies;
 
-/// The size of an individual superblock including trailer.
-pub const superblock_copy_size = @sizeOf(SuperBlockHeader) + superblock_trailer_size_max;
+/// The size of an individual superblock including padding.
+/// TODO Add some padding between copies and include that here.
+pub const superblock_copy_size = @sizeOf(SuperBlockHeader);
```

Comment:
A few reasons to include padding in this number:
- we probably want to reserve a bit of extra space as a just-in-case, since reformatting a whole data file isn't really an option
- can add padding here without requiring an additional zone (`grid_padding`)
- separate physical block addresses (in SSD terms, not grid blocks) of the superblock headers (probably doesn't matter with SSDs :shrug:)